projects/ion/src/lib/modal/models/modal.interface.ts

```diff
@@ -20,6 +21,13 @@ export interface IonModalFooterConfiguration {
   secondaryButton?: IonButtonProps;
 }
 
+interface IonModalHeaderButton {
+  label: string;
```

Comment:
```suggestion
  label?: string;
```
I guess this should be optional as well, since we will possibly have buttons with icon only.